src/components/FieldWrapper.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import {StyleProp, StyleSheet, View, ViewStyle} from "react-native";
+import {Indents} from "../constants/styles";
+
+interface FieldWrapperProps {
+    children: React.ReactNode,
+    indent?: 'sm' | 'md' | 'lg' | 'xl',
+    style?: StyleProp<ViewStyle>,
+
+}
+
+const FieldWrapper = (props: FieldWrapperProps) => {
+    const {children, indent = 'md', style = {}} = props;
+
+    const computedStyle: StyleProp<ViewStyle>[] = [
+        styles.common,
+        indent === 'sm' && styles.indentSm,
```

Comment:
`styles[indent]`

But in this case you need to rewrite keys in `styles`

Reply:
or something like: `marginBottom: styles[Indents.[indent]]`

that you don't need styles at all